_site/README.md

```diff
@@ -0,0 +1,30 @@
+# Welcome!
+
+This is the Study Group Event Reporting tool. It allows users to submit and browse through event reports, closely related the [Mozilla Clubs Events Site](https://github.com/mozilla/clubs-events/).
+
+## This is a prototype
+
+This is a work in progress and uses test data for now.
```

Comment:
Hurray!! https://mozillascience.github.io/studyGroupEvents/ The site pops up with events !! Thank you!